CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### New features
 
++ Add templater support for rendering commit signatures and update default
+  templates.
+
```

Comment:
nit: can be squashed to the templating commit.

We usually include trivial test and doc changes in the implementation patch.